From 0744d6eabe3975c95ba2550a90cbd38d3f49baf3 Mon Sep 17 00:00:00 2001
From: =?utf-8?q?G=C3=A1bor=20Gomb=C3=A1s?= <gombasg@sztaki.hu>
Date: Sat, 29 Dec 2007 16:56:12 +0100
Subject: [PATCH] Fix prototype of service_abort()

All the implementations are defined as returning void, make the prototype
match that.
---
 master/service.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/master/service.h b/master/service.h
index 02a083a..03fef05 100644
--- a/master/service.h
+++ b/master/service.h
@@ -58,7 +58,7 @@ enum {
 extern int service_init(int argc, char **argv, char **envp);
 extern int service_main(int argc, char **argv, char **envp);
 extern int service_main_fd(int fd, int argc, char **argv, char **envp);
-extern int service_abort(int error);
+extern void service_abort(int error);
 
 enum {
     MAX_USE = 250,

